uobtheatre/utils/exceptions.py

```diff
@@ -1,42 +1,78 @@
-from rest_framework.serializers import ValidationError
-from rest_framework.views import exception_handler
+import graphene
 
-from config.settings.common import FIELD_ERRORS_KEY, NON_FIELD_ERRORS_KEY
+# https://gist.github.com/smmoosavi/033deffe834e6417ed6bb55188a05c88
+# TODO We should probably raise these error instead and then format for you in
+# the mutation, ie in mutation mixin wrap the mutation funciton in a try and
+# handle errors there
+# class GQLFieldError(Exception):
+#     """
+#     A single GQL Field error
+#     """
+#
+#     def __init__(self, message, field=None, code=None, params=None):
+#         super().__init__()
```

Comment:
Assuming all these comments should go too?

Reply:
Yeah probably, but there are going to come back I think. The plan is to have all things in a try-catch and then throw these kinds of errors where possible